src/main/java/reposense/parser/ArgsParser.java

```diff
@@ -63,9 +66,14 @@ private static ArgumentParser getArgumentParser() {
 
         // Boolean flags
         parser.addArgument(HELP_FLAGS)
-                .dest(HELP_FLAGS[0])
```

Comment:
Why this removed?

Reply:
Because this is not necessary, the help argument will only execute the action and nothing will be pasted through this flag.